cypress/plugins/index.js

```diff
@@ -49,6 +50,9 @@ module.exports = (on, config) => {
   }
 
   const waitUntilAppRestarts = async (timeout) => await waitOn({ delay: 2000, resources: [config.baseUrl], timeout })
+  const getReplacementText = async (text, source) => source ? fsp.readFile(source) : text
+
+  const makeSureCypressCanInterpretTheResult = () => null
```

Comment:
This function was intended to just replace `undefined` with `null`, I think it's important to pass through other values unless we're sure it makes no difference.

`const makeSureCypressCanInterpretTheResult = (result) => result === undefined ? null : result`

The above code would do this, but if we know it's not needed then there's no point having a more complicated function.

Reply:
I don't think the actual value is required, all we need is to make sure a null is returned.  I'm going to leave it as is.